pkg/executor/import_into.go

```diff
@@ -242,6 +270,57 @@ func (*ImportIntoExec) waitTask(ctx context.Context, jobID int64, task *proto.Ta
 	return err
 }
 
+func (e *ImportIntoExec) getSplitRangeFuncs(ctx context.Context) (
+	addTableSplitRange func(), removeTableSplitRangeAndCloseClients func(), err error) {
+	tidbCfg := config.GetGlobalConfig()
+	tls, err := common.NewTLS(
+		tidbCfg.Security.ClusterSSLCA,
+		tidbCfg.Security.ClusterSSLCert,
+		tidbCfg.Security.ClusterSSLKey,
+		net.JoinHostPort("127.0.0.1", strconv.Itoa(int(tidbCfg.Status.StatusPort))),
+		nil, nil, nil,
+	)
+	if err != nil {
+		logutil.Logger(ctx).Warn("fail to NewTLS", zap.Error(err))
+		return nil, nil, err
+	}
+
+	cfg := e.controller.GetLocalBackendCfg(e.Ctx().GetStore().GetKeyspace(), tidbCfg.Path, "")
+
+	//nolint:forcetypeassert
+	store := e.Ctx().GetStore().(tikv.Storage)
+	pdCli := store.GetRegionCache().PDClient()
+	clients, pdCliForTiKV, err := local.CreateClientsForNewBackend(ctx, tls, cfg, pdCli.GetServiceDiscovery())
```

Comment:
it's not needed for import into from select depends on this part of logic

we can get the PDClient from `store`

Reply:
you can get it from `store.GetCodec` for `EncodeRange`